src/VettedGate.sol

```diff
@@ -34,6 +38,25 @@ contract VettedGate is
 
     mapping(address => bool) internal _consumedAddresses;
 
+    /////////////////////////////////
+    /// Optional referral program ///
+    /////////////////////////////////
+
+    bool public referralProgramActive;
```

Comment:
would suggest to rename it to `isReferralProgramSeasonActive` or among some options with word `ongoing`